src/help.zig

```diff
@@ -48,7 +48,7 @@ const text_with_single_newlines =
     ;
 
 // does nothing to \n\n repeated, but replaces single \n with spaces
-fn removeSingleNewlines(s: []const u8, allocator: *std.mem.Allocator) ![]u8 {
+fn removeSingleNewlines(s: []const u8, allocator: std.mem.Allocator) ![]u8 {
```

Comment:
Is passing the allocator by value surely a good idea? Is it now the recommended way?

Reply:
It is, I just clicked your link